src/api.ts

```diff
@@ -128,13 +128,7 @@ export const resolvers = {
     getGitRepo: async (parent: any, args: {sources: [{repoUrl: string, commit: string}]},
                        context: { onAddRepoRequest: onAddRepoRequestHandler, updateGitLoadingState: loadingStateUpdateHandler }):
         Promise<OperationStatus> => {
-      // If we have a duplicate repo with two commits the actual repo folder is under the first folder.
-      const subDirs = _.map(fs.readdirSync(GIT_ROOT), dir => {
-        if (dir.startsWith(TMP_DIR_PREFIX)) {
```

Comment:
Let's add a piece of code to delete all repos starting with the `TMP_DIR_PREFIX` so we can clean out some of the shite I created

Reply:
![image](https://user-images.githubusercontent.com/42871571/102089053-6342d080-3e24-11eb-8af9-42de512865c2.png)
wouldn't that existing code do the trick?

Reply:
Oh probably